headless-server-lib/src/main/java/com/coremedia/caas/service/repository/content/model/adapter/PageGridAdapter.java

```diff
@@ -12,18 +12,18 @@
   private ContentBackedPageGrid pageGrid;
   private RootContext rootContext;
 
-
   public PageGridAdapter(ContentBackedPageGrid pageGrid, RootContext rootContext) {
     this.pageGrid = pageGrid;
     this.rootContext = rootContext;
   }
 
-
   public String getCssClassName() {
     return pageGrid.getCssClassName();
   }
 
   public List<PageGridPlacementAdapter> getPlacements() {
-    return pageGrid.getPlacements().entrySet().stream().map(e -> new PageGridPlacementAdapter(e.getKey(), e.getValue(), rootContext)).collect(Collectors.collectingAndThen(Collectors.toList(), Collections::unmodifiableList));
+    return pageGrid.getPlacements().entrySet().stream()
+            .map(e -> new PageGridPlacementAdapter(e.getKey(), e.getValue(), rootContext))
+            .collect(Collectors.collectingAndThen(Collectors.toList(), Collections::unmodifiableList));
```

Comment:
Especially here, dropping `Collectors.` via static imports can help the reader to focus on what's really going on.